staging/src/k8s.io/apiserver/pkg/util/flowcontrol/metrics/metrics.go

```diff
@@ -105,67 +104,44 @@ var (
 		},
 		[]string{priorityLevel, flowSchema},
 	)
-	// PriorityLevelExecutionSeatsObserverGenerator creates observers of seats occupied throughout execution for priority levels
-	PriorityLevelExecutionSeatsObserverGenerator = NewSampleAndWaterMarkHistogramsGenerator(clock.RealClock{}, time.Millisecond,
-		&compbasemetrics.HistogramOpts{
+	// PriorityLevelExecutionSeatsObserverVec creates observers of seats occupied throughout execution for priority levels
+	PriorityLevelExecutionSeatsObserverVec = NewTimingRatioHistogramVec(
+		&compbasemetrics.TimingHistogramOpts{
 			Namespace:      namespace,
 			Subsystem:      subsystem,
-			Name:           "priority_level_seat_count_samples",
-			Help:           "Periodic observations of number of seats occupied for any stage of execution (but only initial stage for WATCHes)",
-			Buckets:        []float64{0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1},
+			Name:           "priority_level_seat_utilization",
+			Help:           "Observations, at the end of every nanosecond, of utilization of seats for any stage of execution (but only initial stage for WATCHes)",
```

Comment:
From the perspective of someone who didn't read the KEP or the implementation of P&F, the notion of seats might be a bit vague, could we perhaps mention that these are seats allocated inside of the queue?

Also, renaming the metric to `priority_level_queue_seats_utilization` might make it clearer to the users, wdyt?

Reply:
Difficult issue here. How much documentation to pack into a name?
BTW, the seats are _not_ inside a queue.  All the queues in one QueueSet compete for the same set of execution seats.

Reply:
oh ok, that's something I misunderstood. I think the challenge with these metrics is that they are very complex to understand without a deep-level understanding of the p&f, but might be useful to any users of p&f. Once we've updated the doc to reference the new metrics, I think it would be worth gathering feedback from people outside of the p&f initiative to make sure any k8s users can actually understand how to make use of them, wdyt?

Reply:
Completely agree.  We want the doc as well as the HELP string to make sense to non-experts.